app/controller/sdl/RCModulesController.js

```diff
@@ -354,14 +360,17 @@ SDL.RCModulesController = Em.Object.create({
               break;
             }
             case 'SEAT': {
+              self.seatUUIDs = [];
```

Comment:
@YarikMamykin 
```suggestion
              let seatUUIDs = [];
```

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/8e050fe8f937ae904e6ce7ea0bae7863d2e9bdd7